GitUI/Translation/English.xlf

```diff
@@ -8665,6 +8665,66 @@ help</source>
       </trans-unit>
     </body>
   </file>
+  <file datatype="plaintext" original="JiraCommitHintPlugin" source-language="en">
```

Comment:
This does not belong into this file.

Reply:
But plugin translations shall not be in `English.xlf`. Your modification has a weird effect on the TranslationApp at least - if not at assembly loading at all.
I am little worried because it does not affect all plugins. There seems to even be a random portion. This needs to be analyzed.

Reply:
Is this maybe related to the work where only gitextensions.blaaaa.. and how we did the plugin renaming earlier?

Reply:
The plugin DLL names have not changed since 3.x -- at least in the portable packages.

Why does it affect the JiraCommitHintPlugin only?
Is it the plugin which is finished building first? And the modified assembly loader always loads only the first one found?? 

```
 GitExtensions-Portable-3.6.0.13067-9b938f225\Plugins

14.11.2021  21:18            32.768 AppVeyorIntegration.dll
19.01.2021  04:18           249.856 Atlassian.Jira.dll
14.11.2021  21:18            46.592 AzureDevOpsIntegration.dll
14.11.2021  21:18            13.824 GitExtensions.Plugins.AutoCompileSubmodules.dll
14.11.2021  21:18            17.408 GitExtensions.Plugins.BackgroundFetch.dll
14.11.2021  21:18            65.536 GitExtensions.Plugins.Bitbucket.dll
14.11.2021  21:18            12.800 GitExtensions.Plugins.CreateLocalBranches.dll
14.11.2021  21:24            49.664 GitExtensions.Plugins.DeleteUnusedBranches.dll
14.11.2021  21:18            33.280 GitExtensions.Plugins.FindLargeFiles.dll
14.11.2021  21:18            39.424 GitExtensions.Plugins.GitFlow.dll
14.11.2021  21:18            31.744 GitExtensions.Plugins.GitHub3.dll
14.11.2021  21:18            33.792 GitExtensions.Plugins.GitImpact.dll
14.11.2021  21:18            67.584 GitExtensions.Plugins.GitStatistics.dll
14.11.2021  21:18            29.696 GitExtensions.Plugins.Gource.dll
14.11.2021  21:18            24.576 GitExtensions.Plugins.JiraCommitHintPlugin.dll
14.11.2021  21:18            18.432 GitExtensions.Plugins.ProxySwitcher.dll
14.11.2021  21:18            27.136 GitExtensions.Plugins.ReleaseNotesGenerator.dll
14.11.2021  21:16           200.704 ICSharpCode.SharpZipLib.dll
14.11.2021  21:18            30.720 JenkinsIntegration.dll
17.03.2021  20:03           695.336 Newtonsoft.Json.dll
19.07.2019  09:08            26.112 NString.dll
14.11.2021  21:18            45.056 TeamCityIntegration.dll
```

Reply:
> The plugin DLL names have not changed since 3.x -- at least in the portable packages.

In master, the BuildServer "plugins" appended the "GitExtensions." prefix to the URL so they are loaded in Debug, see #10466.
(Atlassian.Jira.dll and NString.dll for Jira hint and ICSharpCode.SharpZipLib.dll for Gource are loaded from the plugins?)

Reply:
> But plugin translations shall not be in English.xlf. 

As i see, it not only JiraCommitHintPlugin problem. There already were other plugins translations in this file.
But just after my fixes for JiraCommitHintPlugin the existing problem was highlighted.

Reply:
The bug is here:
https://github.com/gitextensions/gitextensions/blob/0df0b9173c609ef102531b0b3002e60cf7d69594/ResourceManager/Xliff/TranslationUtil.cs#L319-L320

A path to a plugin looks something like this, and it will never match:
`C:\dev\gitextensions\artifacts\Debug\bin\GitExtensions\net6.0-windows\Plugins\JiraCommitHintPlugin\GitExtensions.Plugins.JiraCommitHintPlugin.dll`